pkg/pkgmgmt/client/delete.go

```diff
@@ -0,0 +1,44 @@
+package client
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"get.porter.sh/porter/pkg/pkgmgmt"
+	"github.com/pkg/errors"
+)
+
+func (fs *FileSystem) Uninstall(opts pkgmgmt.UninstallOptions) (*pkgmgmt.Metadata, error) {
+	if opts.Name != "" {
+		return fs.uninstallByName(opts.Name)
+	}
+
+	return nil, errors.Errorf("No %s name was provided to uninstall", fs.PkgType)
+}
+
+func (fs *FileSystem) uninstallByName(name string) (*pkgmgmt.Metadata, error) {
+	parentDir, err := fs.GetPackagesDir()
+	if err != nil {
+		return nil, err
+	}
+	pkgDir := filepath.Join(parentDir, name)
+	exists, _ := fs.FileSystem.Exists(pkgDir)
+	if exists == true {
+		err = fs.FileSystem.RemoveAll(pkgDir)
+		if err != nil {
+			return nil, errors.Wrapf(err, "could not remove %s directory %q", fs.PkgType, pkgDir)
+		}
+
+		m := pkgmgmt.Metadata{
+			Name: name,
+			Dir:  parentDir,
+		}
+		return &m, nil
+	}
+
+	if fs.Debug {
+		fmt.Fprintf(fs.Out, "Unable to find requested %s\n", fs.PkgType)
```

Comment:
I don't know whether its big deal or not but we used to prefix debug logs with `DEBUG`

Reply:
I am breaking my own rules, thank you for calling that out. 🙇‍♀

Reply:
After looking at this one further, I think it's fine to not have the DEBUG: prefix. In the other places it was just to help call out that we were dumping variables.

Here it's not needed as we are just printing additional English text when in debug mode.

However I wasn't following the `Err` convention, and the message was lacking, so I improved it.